extensions/extensions-encryption/aissemble-extensions-encryption-vault-python/src/aissemble_encrypt/vault_config.py

```diff
@@ -64,18 +64,23 @@ def secrets_host_url(self):
         Returns the secrets host url.
         This should only be used for integration tests, and not otherwise.
         """
-        return self.properties["secrets.host.url"]
+        if "SECRETS_HOST_URL" in os.environ:
+            # host and port of the aissemble-vault container
+            return os.environ["SECRETS_HOST_URL"]
+        else:
+            return self.properties["secrets.host.url"]
 
     @staticmethod
-    def validate_container_start():
+    def validate_container_start(port):
```

Comment:
I/A: I'm a little skeptical about including this method here in the Config class instead of just in the test classes, but I didn't feel strongly enough to move it.